File: src/main/java/com/tom/Hello.java
package com.tom;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello world");
        String s = "abcde";
        System.out.println(s.length());
        Person p = new Person("Hank",66.2f,1.8f);
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());

       /* p.hello();
        int age = 19;
        Integer age2 = 19;
        float weight = 61.7f;
        Float height = 1.7f;
        char c = 'b';
        Character c2 = 'A';
        byte b = 50;
        boolean adult = true;
        boolean enroll = false;
        String name = "Alan";*/
    }
}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main(args: Array<String>) {
   // print("Hello Kotlin")
    var s : String? = "abcde"
    s = null
    println(s?.substring(2))
    val h  = Human(66.5f,1.7f)
    println(h.bmi())

   /* var age = 19
    age = 20
    var height = 1.7f
    var name : String
    name = "Hank"
    println(name)
    */
}
class Human(var weight : Float,var height : Float ,var name : String = ""){
    init {
        println("test$weight")
    }
//    constructor(name:String,weight: Float,height: Float):this(weight,height)
    fun hello(){
        print("Hello Kotlin")
    }
    fun bmi() : Float{
        val bmi : Float
        bmi = weight/(height*height)
        return  bmi

    }
}